src/atomate2/vasp/flows/mp.py

```diff
@@ -194,6 +200,95 @@ def make(self, structure: Structure, prev_dir: str | Path | None = None) -> Flow
         return Flow(jobs=jobs, output=output, name=self.name)
 
 
+@dataclass
+class MP24DoubleRelaxMaker(DoubleRelaxMaker):
+    """MP24 PBEsol + r2SCAN double relaxation workflow.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    relax_maker1 : .BaseVaspMaker
+        Maker to generate the first relaxation.
+    relax_maker2 : .BaseVaspMaker
+        Maker to generate the second relaxation.
+    """
+
+    name: str = "MP24 double relax"
+    relax_maker1: Maker | None = field(default_factory=MP24PreRelaxMaker)
+    relax_maker2: Maker = field(
+        default_factory=lambda: MP24RelaxMaker(
+            copy_vasp_kwargs={"additional_vasp_files": ("WAVECAR", "CHGCAR")}
+        )
+    )
+
+
+@dataclass
+class MP24DoubleRelaxStaticMaker(Maker):
+    """MP24 workflow to relax a structure with r2SCAN.
+
+    Optionally, files can be automatically cleaned following completion
+    of the workflow. By default, WAVECAR files are removed.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    relax_maker : .BaseVaspMaker
+        Maker to generate the relaxation.
+    static_maker : .BaseVaspMaker
+        Maker to generate the static calculation before the relaxation.
+    clean_files : Sequence of str or None
+        If a list of strings, names of files to remove following the workflow.
+        By default, this removes the WAVECAR files (gzipped or not).
+    """
+
+    name: str = "MP24 r2SCAN workflow"
+    relax_maker: Maker = field(default_factory=MP24DoubleRelaxMaker)
+    static_maker: Maker = field(
+        default_factory=lambda: MP24StaticMaker(
+            copy_vasp_kwargs={"additional_vasp_files": ("WAVECAR", "CHGCAR")}
+        )
+    )
+    clean_files: Sequence[str] | None = ("WAVECAR",)
```

Comment:
One last point: we also delete wavecars in the lobster workflow. Might be worth to make this consistent in the future.